src/v/pandaproxy/schema_registry/api.cc

```diff
@@ -69,7 +71,7 @@ ss::future<> api::start() {
       std::ref(_sequencer),
       std::ref(_controller));
 
-    co_await _service.invoke_on_all(&service::start);
+    co_await _service.invoke_on_all(&service::start, std::ref(_shard0_as));
```

Comment:
I don't think an abort source can be shared between shards. Probably each instance of the service needs an `abort_source`, which can be aborted during stop, probably just before the gate is closed.